### ISSUE TITLE
Auto-update

### ISSUE THREAD

**Issue by lexor90** [pmq20/node-compiler#53]
I think we should write down a small example describing the usage of `auto-update` inside our README.md since only exposing the flags may not be trivial to new users.

@pmq20 may it be possible to better clarify how often it will check the specified URI for updates and maybe make it settable?

**Comment by pmq20** [comment]
Yes. I am working on limiting number of checks to once per day via recording into `~/.libautoupdate`. Development has been moved into https://github.com/pmq20/libautoupdate

I'll update the document once it is stable.
